epoch/api/oracle_api_usage.md

```diff
@@ -13,123 +11,114 @@ to split the example into one node running the oracle and other nodes querying
 that oracle.
 
 The following assumes that the node exposes at address `localhost/127.0.0.1` the following ports:
-* User API internal WebSocket endpoint: 3114 - i.e. connecting to `ws://127.0.0.1:3114/websocket` opens a websocket connection.
-* User API internal HTTP endpoint: 3113.
+* User API external HTTP endpoint: 3013
+* User API internal HTTP endpoint: 3113
+
+## Oracle management flow
 
 In order to work through the example we also need the (Base58Check-encoded)
 public key of the node. This is easily retrieved from the running node:
 ```
-~/epoch/node: curl http://127.0.0.1:3113/v2/account/pub-key
-{"pub_key":"ak$jzZyCLFtHVD7yVdEhGJFM3LjeXrKqWxnHbCYzhnrrR4DkdF"}
+curl http://127.0.0.1:3113/v2/account/pub-key
+
+{"pub_key":"ak$EmJyR97vW4jzdcCPCvgjUa8RUmo45E1KnExBum38yz48Frwov"}
 ```
 
-On a freshly started node, connect a websocket. (Here we use
-[WSCat](https://github.com/websockets/wscat) for the websocket connection.)
-Upon connection we want to subscribe to the _mined_block_-event,
-i.e. everytime the node mines a new block an event is pushed to the client. We
-also need to wait for the first block to be mined by the node (or else our
-transactions will be rejected with _"Insufficient balance"_). In an environment
-with more than one node it is better to subscribe to the _new_block_-event, but
-here we simply use _mined_block_.
+We need to wait for the first block to be mined by the node (or else our
+transactions will be rejected with _"Insufficient balance"_).
 
-```
-~/epoch/node: wscat -c ws://127.0.0.1:3114/websocket
-connected (press CTRL+C to quit)
-> {"target":"chain", "action":"subscribe", "payload":{"type":"mined_block}}
-< {"origin":"chain", "action":"subscribe", "tag":"untagged", "payload":{"result":"ok", "subscribed_to":{"type":"mined_block}}}
-...
-< {"action":"mined_block","origin":"chain","payload":{"height":1,"hash":"bh$jXjgHkcuXnTY4PtMpctcwFT2jf4fZ1jGdeax1geWoW64hSXpY"}}
-```
+### Register
 
 Once the account has a positive balance we can post an _"Oracle register transaction"_:
 ```
-...
-> {"target":"oracle", "action":"register", "payload":{"type":"OracleRegisterTxObject", "vsn":1, "account":"ak$jzZyCLFtHVD7yVdEhGJFM3LjeXrKqWxnHbCYzhnrrR4DkdF", "query_format":"the query spec", "response_format":"the response spec", "query_fee":4, "oracle_ttl":{"type":"delta", "value":50}, "fee":5, "ttl":1234}}
-< {"action":"register","origin":"oracle","tag":"untagged","payload":{"result":"ok","oracle_id":"ok$jzZyCLFtHVD7yVdEhGJFM3LjeXrKqWxnHbCYzhnrrR4DkdF","tx_hash":"th$26iUqaRt4s1ydAF8z7WDeM4FhCqwEu5TbWTCazYFsPY8Le8Upq"}}
+curl -X POST -H "Content-Type: application/json" http://localhost:3113/v2/oracle-register-tx -d '{"type":"OracleRegisterTxObject", "vsn":1, "account":"ak$EmJyR97vW4jzdcCPCvgjUa8RUmo45E1KnExBum38yz48Frwov", "query_format":"the query spec", "response_format":"the response spec", "query_fee":4, "oracle_ttl":{"type":"delta", "value":50}, "fee":5, "ttl":1234}'
```

Comment:
I would suggest refraining from documenting [obsolete paths](https://github.com/aeternity/epoch/blob/dd0da343cd0aa9ab8f4fc23a001430b94775c1ce/config/swagger.yaml#L1760).

Reply:
I used obsolete paths so the example doesn't change too much. Also,  the obsolete paths are used in other docs, too - [spend_api_usage](https://github.com/aeternity/protocol/blob/master/epoch/api/spend_api_usage.md), [naming_system_api_usage](https://github.com/aeternity/protocol/blob/master/epoch/api/naming_system_api_usage.md), etc. These paths are used for posting a tx which is signed by node's key, then included in the chain.

The `/tx/oracle/*` endpoints return an unsigned tx, the client needs to sign it and post using `/tx` endpoint. I haven't investigated how to do signing on the client side.

Anyways, the obsolete endpoints are going to be removed soon and the `tx/oracle/*` are going to be renamed to `/debug/oracles/*` (and eventually removed too as these txs will be created client side only).